client/pages/DataExport/ResultsTable/ResultsTable.tsx

```diff
@@ -0,0 +1,170 @@
+import './ResultsTable.scss'
+import React, { useEffect, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router'
+
+import { Objects } from '@core/utils'
+
+import { Areas, CountryIso } from '@meta/area'
+import { AssessmentName } from '@meta/assessment'
+import { Unit } from '@meta/dataExport'
+
+import { useTableSections } from '@client/store/pages/assessmentSection'
+import { useDataExportCountries, useDataExportSelection } from '@client/store/pages/dataExport'
+import ButtonTableExport from '@client/components/ButtonTableExport'
+import { convertValue, formatValue, getColumnLabelKeys } from '@client/pages/DataExport/utils'
+
+import RowFooter from './RowFooter'
+import Title from './Title'
+import { useFetchResults } from './useFetchResults'
+
+const ResultsTable: React.FC<{ tableName: string }> = ({ tableName }) => {
+  const i18n = useTranslation()
+  const {
+    assessmentName,
+    cycleName,
+    section: assessmentSection,
+  } = useParams<{
+    assessmentName: AssessmentName
+    cycleName: string
+    section: string
+  }>()
+  const selection = useDataExportSelection(assessmentSection)
+  const countries = useDataExportCountries()
+
+  const tableSections = useTableSections({ sectionName: assessmentSection })
+  if (Objects.isEmpty(tableSections)) return null
+  const { tables } = tableSections.find((tableSection) => tableSection.tables.find((table) => table.props.dataExport))
+  const baseUnit = tables[0].props.unit
```

Comment:
why is table[0]? don't we get the table with dataExport flag?

Reply:
you're right